packages/contracts-sdk/src/lib/contracts-sdk.spec.ts

```diff
@@ -1,10 +1,7 @@
 import { LitContracts } from './contracts-sdk';
-import { ethers } from 'ethers';
-import { LitContract } from '../../../types/src/lib/types';
 
 describe('contractsSdk', () => {
   let litContracts: LitContracts;
```

Comment:
Went to update this reference to static rpc provider, but discovered it wasn't actually being used in this test file at all. Removed it